capsules/classifier_safety_gear_openvino/config.py

```diff
@@ -0,0 +1,51 @@
+from vcap import FloatOption
+
+safety_hat = "safety_hat"
+safety_vest = "safety_vest"
+
+safety_gears = [safety_hat, safety_vest]
+
+with_safety_hat = ["without_safe_hat", "with_safe_hat"]
+with_safety_vest = ["without_safe_vest", "with_safe_vest"]
+
+confidence_threshold = "confidence_threshold"
+safety_hat_iou_threshold = "safety_hat_iou_threshold"
+safety_vest_iou_threshold = "safety_vest_iou_threshold"
+
+attributes = {
```

Comment:
I'm not a huge fan of this highly nested structure. 

It leads to really long, complicated lines in the backend like 
```
det.extra_data[attributes[gear_type]["confidence"]] = best_match.extra_data["detection_confidence"]
```

```
det.attributes[attributes[gear_type]["attribute_name"]] = attributes[gear_type]["possible_values"][1]
```

and even 
```
person_det.attributes[attributes[gear_type]["attribute_name"]] = attributes[gear_type]["possible_values"][0]
```

Is there a way to clean this up and simplify the structure, so the code can be more readable in the backend?

Reply:
All I want is to be able to iterate through different gear types so I don't have redundant code. I think I can do something like

`det.attributes[f"{gear_type}"] = f"without_{gear_type}"